### ISSUE TITLE
Is it possible to reuse existing cbindgen configurations for Interoptopus?

### ISSUE THREAD

**Issue by Degot** [ralfbiedert/interoptopus#87]
I'm trying to create C# binding for libsignal. They already have ffi with cbindgen.
Is it possible to reuse it, without going through libsignal code and apply changes for Interoptopus?

**Comment by ralfbiedert** [comment]
No, there is no cbindgen interoperability, Interoptopus pretty much needs the attribute annotations to work.
